.gitignore

```diff
@@ -30,3 +30,4 @@ src/main/resources/adminPassword.txt
 /.apt_generated/
 */plugins/*.jar
 Patient/*
+volume/database/data/
```

Comment:
yeah this change is needed recently faced some build issue after built this data get generated shown in git history to stash it